src/bin/hexyl.rs

```diff
@@ -166,6 +166,17 @@ fn run() -> Result<(), AnyhowError> {
                     based on the current terminal width",
                 ),
         )
+        .arg(
+            Arg::new("group_bytes")
+                .short('g')
```

Comment:
Can we please add a long option for this (`--group-bytes`) and use it in the unit tests?